docs/blog/2013-01-27-refinements101-reax.lhs

```diff
@@ -258,5 +258,5 @@ There are several things to take away.
 [concolic]: http://en.wikipedia.org/wiki/Concolic_testing
 [icse04]: http://goto.ucsd.edu/~rjhala/papers/generating_tests_from_counterexamples.html
 [dsd]: http://dl.acm.org/citation.cfm?doid=1348250.1348254
-[mlton]: http://www.cs.purdue.edu/homes/zhu103/pubs/vmcai13.pdf
+[mlton]: https://www.cs.purdue.edu/homes/suresh/papers/vmcai13.pdf
```

Comment:
The link is dead:

http://www.cs.purdue.edu/homes/zhu103/pubs/vmcai13.pdf

I googled and tried to find an active link. I found this:

https://www.cs.purdue.edu/homes/suresh/papers/vmcai13.pdf

Is that the right paper? Otherwise let me know and I'll adjust this commit.

Reply:
perfect, thank you!